src/status_im/contexts/wallet/router/core.cljs

```diff
@@ -0,0 +1,87 @@
+(ns status-im.contexts.wallet.router.core
+  (:require [schema.core :as schema]
+            [status-im.contexts.wallet.router.schema :as router.schema]
+            [status-im.contexts.wallet.send.transaction-settings.core :as transaction-settings]
+            [status-im.contexts.wallet.send.utils :as send-utils]))
+
+(def ^:private gwei-precision 6)
+
+(defn- to-gwei
+  [amount]
+  (send-utils/convert-to-gwei amount gwei-precision))
```

Comment:
Side note about the name `convert-to-gwei`. This function should ideally live in a separate namespace because it has nothing to do with send utilities. Once the separate namespace exists, it's name will support a better API naming scheme that's more idiomatic. For example, `eth-unit/->gwei` or `eth-money/->gwei`

Reply:
I'm not sure if we need to add another namespace for it, given that we have the `money` one already for dealing with crypto/fiat. At the same time, it seems like `convert-to-gwei` is used only here, so we might not even need an util for it and can just move it to this ns, wdyt?

Reply:
The existing `money` namespace could be used as well as you pointed out. I tend to personally not like to decide where functions should live based on usage, but rather about the purpose of namespaces. The router shouldn't know or have the responsibility of handling crypto conversions directly, so from that angle `convert-to-gwei` feels out of place, even if it's used only by this namespace (for now).